libs/grout/src/server.ts

```diff
@@ -226,6 +230,7 @@ export function buildRouter(
           );
         }
 
+        const durationStart = Date.now();
```

Comment:
not completely sure if the middleware should be included in the timing but I think so since its still part of the time the request takes.